docs/shared/helm-show-router-output.mdx

```diff
@@ -142,6 +139,7 @@ service:
   type: ClusterIP
   port: 80
   annotations: {}
+  targetport: http
```

Comment:
Without working this through totally, I think the ports are no longer wired up correctly. Have you tested that this works? I feel like the `targetport` should now be `4000`. I could be wrong, I just wired it up in my head...